scripts/coccinelle/modules/6.1-i2c_remove_sig.cocci

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Can you add a README.md to `scripts/coccinelle/modules/` that gives some context for this files and explains when and how to use them?

It is not entirely clear from the commit message if all or just some of the patches in "onl: update modules for linux 6.1" have been created by the two semantic patches.

Reply:
Added a (smallish) README.md